Makefile

```diff
@@ -14,4 +14,4 @@ generate: install
 	bash scripts/generate.sh
 
 test-docker: generate
-	bash scripts/prism.sh
+	bash scripts/prism.sh $(SONAR_TOKEN)
```

Comment:
I think this can be `SONAR_TOKEN=$(SONAR_TOKEN) bash scripts/prism.sh` and then you don't need to pass in the `build-arg` option.

Reply:
made the above changes